tests/integration/test_feedback.py

```diff
@@ -139,3 +155,89 @@ def test_feedback_events_without_processing(
 
     userfeedback = envelope.items[0]
     assert userfeedback.type == "feedback"
+
+
+@pytest.mark.parametrize("use_feedback_topic", (False, True))
+def test_feedback_with_attachment_in_same_envelope(
+    mini_sentry,
+    relay_with_processing,
+    feedback_consumer,
+    events_consumer,
+    attachments_consumer,
+    use_feedback_topic,
+):
+    mini_sentry.add_basic_project_config(
+        42, extra={"config": {"features": ["organizations:user-feedback-ingest"]}}
+    )
+    mini_sentry.set_global_config_option(
+        "feedback.ingest-topic.rollout-rate", 1.0 if use_feedback_topic else 0.0
+    )
+    # Test will only pass with this option set
+    mini_sentry.set_global_config_option("feedback.ingest-inline-attachments", True)
+
+    if use_feedback_topic:
+        mini_sentry.set_global_config_option("feedback.ingest-topic.rollout-rate", 1.0)
+        feedback_consumer_ = feedback_consumer(timeout=20)
+        other_consumer = events_consumer(timeout=20)
+    else:
+        mini_sentry.set_global_config_option("feedback.ingest-topic.rollout-rate", 0.0)
+        feedback_consumer_ = events_consumer(timeout=20)
+        other_consumer = feedback_consumer(timeout=20)
+    attachments_consumer_ = attachments_consumer(timeout=20)
```

Comment:
Unless there's a reason to have it:
```suggestion
        feedback_consumer = events_consumer(timeout=20)
        other_consumer = feedback_consumer(timeout=20)
    attachments_consumer = attachments_consumer(timeout=20)
```
(same with the `if` block)

Reply:
Sounds good, I didn't want to shadow/override the fixtures with the same name, but should be fine for this test!

Reply:
Sounds good, I didn't want to shadow/override the fixtures with the same name, but can make this change!